### PR TITLE
[WIP]#4929 [Manager] De-structure the observale of API related from epics

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [ ] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [ ] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [ ] Add Assignees (of course, you are the assignee by default!)
- [ ] DON'T set a Milestone
- [ ] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [ ] Link to an issue(You can do so by using the required by keyword: required by #issue_number)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/store/epics/broker/createBrokerEpic.js [hunk 1]
```diff
@@ -27,15 +27,15 @@ import {
...
+import * as brokerApi from './apiEpic';
 import * as actions from 'store/actions';
 import * as schema from 'store/schema';
 import { getId } from 'utils/object';
 import { LOG_LEVEL, GROUP } from 'const';
 
 export const createBroker$ = values => {
   const brokerId = getId(values);
-  return defer(() => brokerApi.create(values)).pipe(
+  return from(brokerApi.create$(values)).pipe(
```
Comment (saivirtue) [inline ohara-manager/client/src/store/epics/broker/createBrokerEpic.js:38]: The `from()` function could be omitted :)

Comment (wu87988622) [reply ohara-manager/client/src/store/epics/broker/createBrokerEpic.js:38]: Thanks, you are rigth

File: ohara-manager/client/src/store/epics/broker/apiEpic.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import * as brokerApi from 'api/brokerApi';
```
Comment (eechih) [inline ohara-manager/client/src/store/epics/broker/apiEpic.js:1]: @wu87988622 在我們的專案中，`Epic` 應該是指那些會被 Redux-Observable 直接載入執行的 Observable，然而這些 api 是屬於一般的 Observable，我個人覺得 `apiEpic` 不太合適。

@konekoya @saivirtue  大家有何看法？

Comment (saivirtue) [reply ohara-manager/client/src/store/epics/broker/apiEpic.js:1]: 可以改個名稱?  像是`apiObservables` ?

Comment (konekoya) [reply ohara-manager/client/src/store/epics/broker/apiEpic.js:1]: I'm just wondering if we really need this abstraction... But + 1 for not naming it with the keyword "epic".

**Comment by wu87988622** [comment]
優先處理#5139. 先關閉此PR

**Pull request closed without merging**